tests/test_cli.py

```diff
@@ -1,48 +1,63 @@
+import os
+
 from freezegun import freeze_time
 
 from llama.cli import cli
+from llama.s3 import S3
 
 
-@freeze_time("2021-01-01")
-def test_concat_timdex_export(s3_session, bucket_env, runner):
-    # today's date
-    assert len(s3_session.client("s3").list_objects(Bucket="ils-sftp")["Contents"]) == 4
-    assert "Contents" not in s3_session.client("s3").list_objects(
-        Bucket="dip-ils-bucket"
-    )
+def test_concat_timdex_export_all_options_provided_success(mocked_s3, runner):
+    s3 = S3()
```

Comment:
Very minor but even though this is 1 line,  it's called 12 times and could be a fixture and added as an argument to all the tests to tighten up the code